src/Altinn.Notifications.Persistence/Migration/v0.03/01-setup-functions.sql

```diff
@@ -0,0 +1,17 @@
+DROP FUNCTION IF EXISTS notifications.getorders_pastsendtime_updatestatus();
+CREATE OR REPLACE FUNCTION notifications.getorders_pastsendtime_updatestatus()
```

Comment:
i guess v0.02 and v0.03 can be merged to one version @tba76

Reply:
yes